dotalytics_api/main.py

```diff
@@ -103,43 +107,43 @@ def get_matches(account_id: str, matches_requested: int = 5):
                     backpack_2_name = item.name
 
             for hero in heroes.heroes:
-                if hero.id == matches[m].players[p].hero_id:
-                    matches[m].players[p] = player.Player(
-                        player_slot=matches[m].players[p].player_slot,
-                        team_number=matches[m].players[p].team_number,
-                        hero_id=matches[m].players[p].hero_id,
-                        item_0=matches[m].players[p].item_0,
-                        item_1=matches[m].players[p].item_1,
-                        item_2=matches[m].players[p].item_2,
-                        item_3=matches[m].players[p].item_3,
-                        item_4=matches[m].players[p].item_4,
-                        item_5=matches[m].players[p].item_5,
-                        item_neutral=matches[m].players[p].item_neutral,
-                        backpack_0=matches[m].players[p].backpack_0,
-                        backpack_1=matches[m].players[p].backpack_1,
-                        backpack_2=matches[m].players[p].backpack_2,
-                        item_netral=matches[m].players[p].item_neutral,
-                        kills=matches[m].players[p].kills,
-                        deaths=matches[m].players[p].deaths,
-                        assists=matches[m].players[p].assists,
-                        leaver_status=matches[m].players[p].leaver_status,
-                        last_hits=matches[m].players[p].last_hits,
-                        denies=matches[m].players[p].denies,
-                        gold_per_min=matches[m].players[p].gold_per_min,
-                        xp_per_min=matches[m].players[p].xp_per_min,
-                        level=matches[m].players[p].level,
-                        net_worth=matches[m].players[p].net_worth,
-                        aghanims_scepter=matches[m].players[p].aghanims_scepter,
-                        aghanims_shard=matches[m].players[p].aghanims_shard,
-                        moonshard=matches[m].players[p].moonshard,
-                        hero_damage=matches[m].players[p].hero_damage,
-                        tower_damage=matches[m].players[p].tower_damage,
-                        hero_healing=matches[m].players[p].hero_healing,
-                        gold=matches[m].players[p].gold,
-                        gold_spent=matches[m].players[p].gold_spent,
-                        scaled_hero_damage=matches[m].players[p].scaled_hero_damage,
-                        scaled_tower_damage=matches[m].players[p].scaled_tower_damage,
-                        scaled_hero_healing=matches[m].players[p].scaled_hero_healing,
+                if hero.id == p.hero_id:
+                    p = player.Player(
```

Comment:
The matches object is no longer being updated with the new player information. hero_name for one is missing from the response object.